### ISSUE TITLE
Articles for community Discord servers

### ISSUE THREAD

**Issue by Walavouchey** [ppy/osu-wiki#6904]
This has been on my idea list for a while (and probably has been discussed somewhere, not entirely sure). Currently there's only the [osu!dev article](https://osu.ppy.sh/wiki/Community/osu%21dev_Discord_server).

The essential idea is to briefly describe what the servers are about and perhaps explain their channels/channel systems (just like the osu!dev article)

Not entirely sure if any random server should be listed, but I've tried to write down the major ones for each part of the community. Most of these are linked in the `#resources` channel of the osu! server (and they often link to each other). Comment below if you think I've missed any.

### Community / Misc

- [ ] [osu!](https://discord.gg/0Vxo9AsejDkGlk3H)
- [ ] [Aiess Project](https://discord.gg/2XV5dcW) (just various feed channels; debatable)

### Gameplay

- [ ] [osu! University](https://discord.gg/QubdHdnBVg)

### Dev

- [ ] [Performance Points](https://discord.gg/aqPCnXu)

### Mapping and modding

- [ ] [Mapset Management Server](https://discord.gg/8BquKaS)
- [ ] [Cozy Mappers' Café](https://discord.gg/uDjwjaZ)
- [ ] [osu!catch Modding and Mapping Hub](https://discord.gg/ZuxFc4q)
- [ ] [osu!taiko Mapping and Modding Hub](https://discord.gg/yRjvvyZ)
- [ ] [osu!mania Mapping & Modding Hub](https://discord.gg/FqbDdYN)

### Skinning

- [ ] [skinship](https://discord.gg/AZ79xJb)

### Storyboarding

- [ ] [osu! storyboarder banquet](https://discord.gg/B8NX7YW)

**Comment by huuishuu** [comment]
You could also add the osu!taiko & osu!mania Project Loved hubs to this list as well.

**Comment by Rekunan** [comment]
You should also add the [mania diffcalc server](https://discord.gg/GFCNNg8bwk)

**Comment by ZeroPyrozen** [comment]
This could be expansion of `Community/Organisations` section so it also cover online communities too. The article itself can follow this format

```
# Community Name

Short description about this Discord community, might add brief history here. 

## Staffs

| User | Role |
| :-- | :-- |
| [BanchoBot](https://osu.ppy.sh/users/3) | RoleName |
| [...](https://osu.ppy.sh/users/3) | ... |

## Links

- **[Discord server](https://discord.gg/ppy)**
- [Website If Exist](https://osu.ppy.sh)

## Community Rules
- Rule 1
- Rule 2

## Channels

| Channel | Description |
| :-- | :-- |
| `#tournaments` |  Discussing official and officially supported tournaments in the osu! community of all kinds! |

## Activities

Activities that happen in community could be recurring or one time, e.g. mapping contest.
```

**Comment by ZeroPyrozen** [comment]
[osu! Medal Hunters](https://discord.gg/osumedalhunters)

**Comment by Walavouchey** [comment]
> This could be expansion of `Community/Organisations` section so it also cover online communities too

I like the formatting idea, but I'm not sure about calling them organisations or communities. The thing is that a Discord server doesn't always equal community (the Aiess Discord (feed channels only), osu!dev and some other dev servers (dev community??)), and it would be better to keep the listing in one place than to separate them due to categorising. The "communities" here also overlap (with some servers being bigger than others), so having a list of (separate) "communities" wouldn't make that much sense

**Comment by cl8n** [comment]
"Organisations" was intended to be for clubs established offline. their purpose & activities are usually very different from online community

**Comment by peppy** [comment]
Am very neutral and non-committal on this happening.

But if it does, there needs to be stringent standards in place for the servers which are added. Else it's going to become a shitfest of every man and his camel submitting their server for addition. At very least some kind of moderation guarantee and a minimum user count before being accepted.

**Comment by RockRoller01** [comment]
So what is the plan with this one going forward? Stringent standards are of course a must, but what do you have in mind exactly @peppy ?

**Comment by Walavouchey** [comment]
The servers listed in OP should be a good start, even if the specific requirements for later additions are ironed out later.

**Comment by peppy** [comment]
I agree with the list in the opening post. Any further additions can be checked by me or someone else on the team.

**Comment by TicClick** [comment]
individual pages should be converted into small sections of the main page instead once https://github.com/ppy/osu-wiki/issues/8194 is completed

**Comment by Albionthegreat** [comment]
I would like to suggest adding 2 tournament related discord servers to the listing.

- [osu! Tournament Hub](https://discord.gg/bvhajDC)
A server dedicated to helping people with anything and everything related to osu! tournaments. From help with spreadsheets or technical things such as lazer, to questions regarding graphics and tournament hosting, this is a server intended to help you out. You may also find new tournaments to help with or advertise your own tournaments here in case you still need staff. With almost 1500 members, it's quite a large server. It's also run by respected members of the community and boasts a huge list of community resources made by various members of the community.

- [osu!mania Tourney Central](https://discord.gg/WnMcrUnGV5)
Similar in intent to the osu! Tournament Hub server, but this time dedicated to osu!mania tournaments specifically, this server provides a place to discuss the osu!mania world cups, as well as advertise and find new community tournaments for the mania game mode. It's not as large as the tournament hub, but with almost 750 members and a skilled team of respected tournament staffers running the show, it's a good place to go if you want to become more involved in osu!mania tournaments.

**Comment by MChecaH** [comment]
Suggesting bringing up [osu! Game](https://discord.gg/osu) as it has been brought up by clayton. By far one of the biggest communities for all game modes. Mainly used to discuss gameplay from all four modes, holding occasional events as well.

**Comment by cl8n** [comment]
if u r just adding more then one last one I would recommend is osu!alternative https://discord.gg/VZWRZZXcW4 , this one has tools for tracking scores and ranking players/scores by a lot of other metrics you don't see in osu officially

after that and the servers mentioned above, I think that's a good stopping point, at least based on the servers im aware of. there are more public servers ofc but I can't think of any that are very general like the others here so far
